integration/web3js/index.js

```diff
@@ -17,8 +17,11 @@ let userAccount = web3.eth.accounts.privateKeyToAccount("0xf6d5333177711e562cabf
 (async function () {
 
     // get the latest block
-    let block = await web3.eth.getBlockNumber()
-    console.log("🎉 Latest block is:", block)
+    let blockNumber = await web3.eth.getBlockNumber()
```

Comment:
are these changes relevant?

Reply:
Yeah, so that the JS script checks the relevant endpoint for retrieving blocks.